Modules/ModParser-3_0.lua

```diff
@@ -2069,6 +2073,7 @@ local specialModList = {
 	["added small passive skills have (%d+)%% increased effect"] = function(num) return { mod("JewelData", "LIST", { key = "clusterJewelIncEffect", value = num }) } end,
 	["this jewel's socket has (%d+)%% increased effect per allocated passive skill between it and your class' starting location"] = function(num) return { mod("JewelData", "LIST", { key = "jewelIncEffectFromClassStart", value = num }) } end,
 	-- Misc
+	["warcries exert (%d+) additional attack"] = function(num) return { mod("ExtraExertedAttacks", "BASE", num) } end,
```

Comment:
This mod should say:
["warcries exert (%d+) additional attacks?"]
Just in case there are more sources of this mod in the future that give more than 1 extra exertion, this will make it work without needing changed at a later date.